q2_types/sample_data/tests/test_transformer.py

```diff
@@ -26,6 +26,10 @@ def test_pd_series_to_alpha_diversity_format(self):
                         name='shannon', index=exp_index)
 
         obs = transformer(exp)
+        '''
+        Squeeze equals true in function call below to convert single column
```

Comment:
Is it converting a dataframe to a series, or is it returning a series instead of a dataframe? Both are basically the same, but indicating that we are changing the return type will be really helpful for readers of this comment.